### PR TITLE
[Doc] AVA action and MVTec data formats

### PR DESCRIPTION
**Pull request by wonjuleee**
### Checklist

- [ ] I submit my changes into the `develop` branch
- [ ] I have added description of my changes into [CHANGELOG](https://github.com/openvinotoolkit/datumaro/blob/develop/CHANGELOG.md)
- [x] I have updated the [documentation](
  https://github.com/openvinotoolkit/datumaro/tree/develop/docs) accordingly
- [ ] I have added tests to cover my changes
- [ ] I have [linked related issues](
  https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword)

### License

- [ ] I submit _my code changes_ under the same [MIT License](
  https://github.com/openvinotoolkit/datumaro/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.
- [ ] I have updated the license header for each file (see an example below)

```python
# Copyright (C) 2023 Intel Corporation
#
# SPDX-License-Identifier: MIT
```

### CODE REVIEW

File: site/content/en/docs/formats/ava_action.md [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+   ├── frames/
+      ├── video0/ # directory with list of frames extracted from video0
+      │   ├── img1.jpg
+      |   ├── img2.jpg
+      |   └── ...
+      ├── video1/ # directory with list of frames extracted from video1
+      │   ├── img1.jpg
+      |   ├── img2.jpg
+      |   └── ...
+   └── annotations/
```
Comment (vinnamkim) [inline site/content/en/docs/formats/ava_action.md]:
```suggestion
   ├── frames/
   │  ├── video0/ # directory with list of frames extracted from video0
   │  │   ├── img1.jpg
   │  |   ├── img2.jpg
   │  |   └── ...
   │  └── video1/ # directory with list of frames extracted from video1
   │      ├── img1.jpg
   │      ├── img2.jpg
   │      └── ...
   └── annotations/
```

File: site/content/en/docs/formats/ava_action.md [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+```
+
+It is possible to specify project name and project directory. Run
+`datum create --help` for more information.
+
+The AVA action dataset directory should have the following structure:
+
+<!--lint disable fenced-code-flag-->
+```
+└─ Dataset/Category
```
Comment (vinnamkim) [inline site/content/en/docs/formats/ava_action.md]: Just `Dataset/` ?

File: site/content/en/docs/formats/ava_action.md [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+    -- \
+    --save-media
+```
+
+Extra options for exporting to AVA action format:
+- `--save-media` - allow to export dataset with saving media files
+  (by default `False`).
+
+```bash
+datum export -f ava --
```
Comment (vinnamkim) [inline site/content/en/docs/formats/ava_action.md]:
```suggestion
datum export -f ava -- --extra-option-1 --extra-option-2 ...
```

File: site/content/en/docs/formats/mvtec.md [hunk 4]
```diff
@@ -0,0 +1,141 @@
...
+and it turns into bounding boxes or abnormal labels for supporting classification,
+detection, and segmentation tasks. The MVTec AD dataset is composed of training data only
+for `good` category without any annotation and testing data for both `good` and multiple
+`defective` categories with masks. The dataset contains total 15 kinds of objects or textures.
+
+Supported tasks / formats:
+- The combined format - `mvtec`
+- Image classification - `mvtec_classification`
+- Object detection - `mvtec_detection`
+- Class and segmentation - `mvtec_segmentation`
```
Comment (vinnamkim) [inline site/content/en/docs/formats/mvtec.md]: Do we have to rename `Class and segmentation` to `Instance segmentation`?

File: site/content/en/docs/formats/mvtec.md [hunk 5]
```diff
@@ -0,0 +1,141 @@
...
+      |   └── ...
+   └── ground_truth/ # directory with semantic segmentation masks
+      ├── defective1/ # directory with list of defective images for detection and segmentation task
+      │   ├── img1_mask.png
+      |   ├── img2_mask.png
+      |   └── ...
+      ├── defective2/ # directory with list of defective images for detection and segmentation task
+      │   ├── img1_mask.png
+      |   ├── img2_mask.png
+      |   └── ...
```
Comment (vinnamkim) [inline site/content/en/docs/formats/mvtec.md]:
```suggestion
└─ Dataset
   ├── train/
   |   ├── good/ # directory with list of good images
   |   │   ├── img1.png
   |   |   ├── img2.png
   |   |   └── ...
   ├── test/
   |   ├── good/ # directory with list of good images
   |   │   ├── img1.png
   |   |   ├── img2.png
   |   |   └── ...
   |   ├── defective1/ # directory with list of defective images
   |   │   ├── img1.png
   |   |   ├── img2.png
   |   |   └── ...
   |   └── defective2/ # directory with list of defective images
   |       ├── img1.png
   |       ├── img2.png
   |       └── ...
   └── ground_truth/ # directory with semantic segmentation masks
       ├── defective1/ # directory with list of defective images for detection and segmentation task
       │   ├── img1_mask.png
       |   ├── img2_mask.png
       |   └── ...
       └── defective2/ # directory with list of defective images for detection and segmentation task
           ├── img1_mask.png
           ├── img2_mask.png
           └── ...
```

File: site/content/en/docs/formats/ava_action.md (hunk 2 above)
Comment (wonjuleee) [reply site/content/en/docs/formats/ava_action.md]: Yes, that is mvtec indeed.

File: site/content/en/docs/formats/ava_action.md (hunk 1 above)
Comment (wonjuleee) [reply site/content/en/docs/formats/ava_action.md]: I corrected

File: site/content/en/docs/formats/mvtec.md (hunk 4 above)
Comment (wonjuleee) [reply site/content/en/docs/formats/mvtec.md]: I updated. Thanks.

File: site/content/en/docs/formats/ava_action.md (hunk 2 above)
Comment (vinnamkim) [reply site/content/en/docs/formats/ava_action.md]: Does AVA format also has its category name at the top level directory?

Comment (wonjuleee) [reply site/content/en/docs/formats/ava_action.md]: I am sorry that I misunderstood. I will correct this.

**Review by vinnamkim — APPROVED** [review]
LGTM.

**Review by sooahleex — APPROVED** [review]
Looks good to me!

**Pull request merged**